pkgs/development/python-modules/pyjson/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchzip
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "pyjson";
+  version = "1.4.5";
+  pyproject = true;
+
+  src = fetchzip {
+    url = "https://github.com/leeyoshinari/Small_Tool/releases/download/${version}/pyjson-${version}.tar.gz";
+    sha256 = "sha256-KF03x5O8hJICdd2RNMmTCEVl19tcVSR4fYta3Sue9e8=";
+  };
```

Comment:
Use `fetchFromGitHub` and `sourceRoot` instead. That way we get a fetcher that supports automatic updates.